README.md

```diff
@@ -18,7 +18,7 @@ For a dynamic language like Ruby, the build-time and run-time environments are t
 1. The container process transforms that source code into the appropriate runnable setup - in this case, by installing dependencies with Bundler and moving the source code into a directory where Apache has been preconfigured to look for the Ruby `config.ru` file.
 1. Commit the new container and set the image entrypoint to be a script (provided by the builder image) that will start Apache to host the Ruby application.
 
-For compiled languages like Java, C, or Golang, the dependencies necessary for compilation might dramatically outweigh the actual runtime artifacts. To keep runtime images slim, S2I enables multiple-step build processes, where a binary artifact such as an executable or Java WAR file is created in the first builder image, extracted, and injected into an second image that simply places the executable in the correct location for execution.
+For compiled languages like Java, C, or Golang, the dependencies necessary for compilation might dramatically outweigh the actual runtime artifacts. To keep runtime images slim, S2I enables a multiple-step build processes, where a binary artifact such as an executable or Java WAR file is created in the first builder image, extracted, and injected into a second runtime image that simply places the executable in the correct location for execution.
```

Comment:
Its funny that this paragraph seems to be copy-pasted into `docs/runtime_image.md`, but the contents are not exactly the same. In particular, there seems to be an arbitrary difference in the programming languages mentioned:

```
For compiled languages like {+C/C++,+} Java,[-C,-] or Golang, the dependencies
necessary for compilation might dramatically outweigh the {+size of the+} actual
runtime [-artifacts.-]{+artifacts, or provide attack surface areas that are
undesirable in an application image.+} To keep runtime images slim, S2I enables
a multiple-step build processes, where a binary artifact such as an executable
or Java WAR file is created in the first builder image, extracted, and injected
into [-a-]{+an+} second[-runtime-] image that simply places the executable in
the correct location for execution. {+To give you even more abilities to
customize a resulting image, S2I is also executing an `assemble-runtime` script
inside of this (runtime) image. In this way you may do final adjustments by
modifying files before an image will be committed.+}
```

(output from `git diff --word-diff`)